frontend/src/app/registration/page.tsx

```diff
@@ -22,21 +25,28 @@ const Registration = () => {
     }));
   };
 
-  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
-    console.log('Form submitted:', formData);
-    // Add your registration logic here
+
+        await fetch('http://localhost:5156/user', {
```

Comment:
We need to change this to the deployed backend url. Also "user" starts with capital "User".